pkg/migration-tool/reader/reader.go

```diff
@@ -81,12 +79,12 @@ func (rr *RemoteRead) Run(errChan chan<- error) {
 				return
 			}
 			ms := []*labels.Matcher{labels.MustNewMatcher(labels.MatchRegexp, labels.MetricName, ".*")}
-			result, err = blockRef.Fetch(rr.c, rr.client, blockRef.Mint(), blockRef.Maxt(), ms)
+			containsTs, err = blockRef.Fetch(rr.c, rr.client, blockRef.Mint(), blockRef.Maxt(), ms)
 			if err != nil {
 				errChan <- fmt.Errorf("remote-run run: %w", err)
 				return
 			}
-			if len(result.Timeseries) == 0 {
+			if !containsTs {
```

Comment:
why not `len(blockRef.Timeseries) == 0`

Reply:
Same as https://github.com/timescale/promscale/pull/400#discussion_r551758175.

Reply:
again we don't need a result here. we need `len(blockRef.timeseries) == 0` or am I missing something?

Reply:
Yes you are right. That way, it would be efficient. I have updated it accordingly. Thank you very much.